### ISSUE TITLE
Mars arena still allow to escape from it vertically

### ISSUE THREAD

**Issue by AleksNesterzz** [ValveSoftware/Dota2-Gameplay#1922]
### Ability name

Arena of blood

### Description

If arena's spear damage was triggered firstly, you can escape from it by using pounce, ursa's 1st spell, mirana's leap, even rolling boulder of earth spirit and toss of tiny. In addition you can't escape arena using firefly or DK ultimate 18th lvl with aghs, but you can escape by night stalker's ultimate and wyvern 1st spell. Is it supposed to work like this or did i just misunderstand the 7.31 patchnote string "Can no longer be escaped vertically"?

### Example Match ID (and possibly Timestamp)

6676454399 7:34

**Comment by shaileshgarg** [comment]
#241

**Comment by DipSwitch** [comment]
Probably also allowed to escape using **Solar Guardian** of **Dawnbreaker** but haven't tried it yet.

**Comment by yckmm** [comment]
> did i just misunderstand the 7.31 patchnote string "Can no longer be escaped vertically"?
Yes, although the patch note was not worded very well. What got fixed is Mars spearing heroes out of arena that are affected by abilities that displace vertically (Tiny toss, Lion stun, Kunkka torrent etc.). Before the heroes would fly over the wall and out of the arena, now they will be pinned to the arena wall.

> Probably also allowed to escape using **Solar Guardian** of **Dawnbreaker** but haven't tried it yet.
Of course Dawnbreaker ult can escape arena, why wouldn't it.
